.github/workflows/codeql.yml

```diff
@@ -6,16 +6,19 @@ name: "CodeQL"
 
 on:
   pull_request:
-    branches: [main]
+    branches:
+      - main
+      - v[0-9]+-stable
```

Comment:
So far looks good. Just don't forget to change branches to match our source-git branches.